Application/Relay/can_relay.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * can_relay.cpp
+ *
+ *  Created on: May 16, 2024
+ *      Author: Nico
+ */
+#include "./can_relay.hpp"
+namespace application{
+
+	Can_Relay::Can_Relay(std::shared_ptr<platform::ICan> can_bus,
+						 CircularQueue<DataPayload> queue):
+		can_bus_(can_bus),
+		queue_(queue)
+		{ messageSize = queue_.GetSize(); }
```

Comment:
tiny suggestion: new line

Reply:
I think this was the autoformatter, in any case fixed